tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38, py39, py310, pylint, format, flake8, docs
+envlist = py38, py39, pylint, format, flake8, docs
```

Comment:
Why remove py310?  I am testing against those three for pip.  Hopefully the `skip_missing_interpreters = true` is skipping this and not erroring out for you.

Reply:
The `py310` case was the only failing test when I ran a `tox -r`.
I assumed (without confirming) that the `intel-numpy` package is incompatible w/ Python 3.10.

Reply:
It pass on OS X, I'll get to windows when I can.

```
.venv-numpy/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta
  py38: OK (103.21=setup[28.20]+cmd[75.01] seconds)
  py39: OK (85.21=setup[19.21]+cmd[65.99] seconds)
  py310: OK (77.73=setup[16.37]+cmd[61.37] seconds)
  congratulations :) (266.29 seconds)
```

Reply:
I'm pushing a commit to add back py310.